testing/test_parseopt.py

```diff
@@ -125,6 +125,14 @@ def test_parse2(self, parser: parseopt.Parser) -> None:
         args = parser.parse([Path(".")])
         assert getattr(args, parseopt.FILE_OR_DIR)[0] == "."
 
+    @pytest.mark.filterwarnings("ignore:'encoding' argument not specified")
```

Comment:
```suggestion
    # Warning ignore because of:
    # https://github.com/python/cpython/issues/85308
    # Can be removed once Python<3.12 support is dropped.
    @pytest.mark.filterwarnings("ignore:'encoding' argument not specified")
```